multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/Messages.java

```diff
@@ -490,7 +491,7 @@ public class Messages {
     public static final String REMAINING_SERVICES_TO_POLL = "Remaining services to poll: {0}";
     public static final String LAST_OPERATION_FOR_SERVICE = "Last operation for service \"{0}\": {1}";
     public static final String DELETING_ERROR_TYPE_O_FOR_PROCESS_1 = "Deleting error type \"{0}\" for process \"{1}\"";
-    public static final String UPLOAD_APP_TIMEOUT = "Upload app timeout in seconds: {0}";
+    public static final String TIMEOUT_MESSAGE = "Application {0} timeout is set by \"{1}\" parameter to {2} seconds";
```

Comment:
For default value introduce other message:
Application stage timeout is default one: 3,600 seconds

Otherwise the log message migh introduce misleading that there is parameter called "default":
Application stage timeout is set by "default" parameter to 3,600 seconds